tests/modules/organization/fixture/variables.tf

```diff
@@ -15,93 +15,76 @@
  */
 
 variable "custom_roles" {
-  type    = map(list(string))
+  type    = any
```

Comment:
is this intentional?

Reply:
yep, we don't need to specify var types in test fixtures

Reply:
Thank you